msg/MotionPlanRequest.msg

```diff
@@ -47,3 +47,8 @@ float64 allowed_planning_time
 float64 max_velocity_scaling_factor
 float64 max_acceleration_scaling_factor
 
+# Maximum cartesian speed for the given end effector.
+# For the speed only values larger than 0 are allowed.
+# For values outside the range, the trajectory is not modified.
```

Comment:
what range? what values?

Reply:
I changed the line to clarify that the trajectory is not modified when max_cartesian_speed <= 0.